src/lib/eth-calls.ts

```diff
@@ -12,10 +12,12 @@ async function registerTable(
   const address = await signer.getAddress();
   /* eslint-disable-next-line camelcase */
 
-  const contractAddress = this.contract ?? contractAddresses[this.network];
+  const contractAddress = this.contract || contractAddresses[this.network];
```

Comment:
The issue was using the `??` which resulted in a cryptic error message saying 'network does not support ENS'. The value of `this.contract` was potentially an empty string which results in `contractAddress` being an empty string, which in turn is interpreted by ethersproject as an ENS domain.